.golangci.yaml

```diff
@@ -44,3 +44,28 @@ linters-settings:
       - standard
       - default
       - prefix(github.com/openshift/cluster-monitoring-operator)
+
+issues:
+  exclude-rules:
+    - path: _test\.go
+      linters:
+      # Only enable the following linters for test files (commented out):
```

Comment:
we can remove these comments I think, the list of linters above can be the source of truth.